src/base.ts

```diff
@@ -0,0 +1,12 @@
+// Aggregate all the tslint rules
+// See https://palantir.github.io/tslint/rules/
+
+import * as merge from 'merge';
+import {functionalityRules} from './rules/functionality';
+import {maintainabilityRules} from './rules/maintainability';
+import {styleRules} from './rules/style';
+import {typescriptRules} from './rules/typescript';
+
+module.exports = {
+  'rules': merge(functionalityRules, maintainabilityRules, styleRules, typescriptRules),
```

Comment:
This is utilizing es6's spread operator, correct?
Which I presume is fully supported in node 6.9.5 and up.